mmseg/models/decode_heads/hamburger_head.py

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
The original repo: https://github.com/Gsunshine/Enjoy-Hamburger

Reply:
Ok